include/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+###############################################################################
+# Subdirectories
+###############################################################################
+
+add_subdirectory(ueye_cam)
+
+###############################################################################
+# Files
+###############################################################################
+
+file(GLOB HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.hpp)
```

Comment:
I think the use of `file(GLOB)` is usually discouraged these days.

Reply:
Oh, aye. Copied from an old ROS1 codebase where we were pragmatic about it when managing 100+ packages - time lost chasing newly added, but missing headers in binary installations was costly.

Not a problem for us here, updated.